edgedb/asyncio_pool.py

```diff
@@ -579,33 +528,38 @@ def __init__(self, *connect_args,
         )
 
     @property
-    def min_size(self) -> int:
-        """Number of connection the pool was initialized with."""
-
-        return self._impl._minsize
-
-    @property
-    def max_size(self) -> int:
+    def concurrency(self) -> int:
         """Max number of connections in the pool."""
 
-        return self._impl._maxsize
+        return self._impl._concurrency
 
     @property
     def free_size(self) -> int:
         """Number of available connections in the pool."""
 
         if self._impl._queue is None:
             # Queue has not been initialized yet
-            return self._impl._maxsize
+            return self._impl._concurrency
 
         return self._impl._queue.qsize()
 
+    async def ensure_connected(self):
+        for ch in self._impl._holders:
+            if ch._con is not None and ch._con.is_closed():
```

Comment:
Just to be sure - this is not intentionally `and ch._con.is_closed()` here for some reason, right? Because I think it's a typo, it should be `and not ch._con.is_closed()`.

Reply:
Yea, it's a typo